telicent_lib/adapter.py

```diff
@@ -207,7 +207,9 @@ def run(self) -> None:
                             default_headers = [
                                 ('Exec-Path', self.generated_id),
                                 ('Request-Id', request_id),
-                                ('traceparent', carrier.get('traceparent', ''))
+                                ('traceparent', carrier.get('traceparent', '')),
+                                ('Data-Source-Name', str(self.source_name)),
```

Comment:
See Rob's comments (on ticket) about being it better to show nothing than None if there's no value

Reply:
This also applies to lines 51, 52, 97 and 98 in test_adapter.py where the default values for Data-Source-Name and Data-Source-Type appear to set to b'None'.